### DISCUSSION TITLE
Should custom fields be shown in the address card on the order page in the CP ?

### DISCUSSION THREAD

**Discussion by MisterMike** [craftcms/commerce#3054] [Ideas] [answered]
One can add custom fields to the Address field layout, and they'll show up if you edit them, but they don't show up in that little card view on top of the order detail page in the control panel. 

As Brad mentioned to me, this might related to UI complexity, but I have a situation where the client would like to show the customer's phone number in that Address card – like Commerce 3 did.

Might there be a way to allow the selection of certain custom fields to be included in the Address block ? What about the same process like with the list of allowed countries, where you can drag/select tags into a field that looks like a textarea ?

**Comment by lenvanessen** [comment]
Yes, 100% agree. We store the houseNumber in a separate field, so for us the address is incomlete. An event in Crafts Addresses service method `formatAddress` would be sufficient, as we could just add a custom formatter there.

**Comment by jmauzyk** [comment]
I agree, this would be a helpful feature as the only way to view the phone number right now would be to edit the order, then edit the address, which isn't immediately apparent to users.

**Comment by lukeholder** [comment, marked as answer]
Craft 4.5 added the ability to replace the default address formatter. 


https://craftcms.com/docs/4.x/addresses.html#customizing-the-formatter

See at the bottom of that section.
